### PR TITLE
add test case for EncryptDQLResultDecorator

### PR DESCRIPTION
**Pull request by dhc575757**
add test case for EncryptDQLResultDecorator #6864

### CODE REVIEW

File: shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-merge/src/test/java/org/apache/shardingsphere/encrypt/merge/dql/EncryptDQLResultDecoratorTest.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import org.apache.shardingsphere.sql.parser.binder.statement.SQLStatementContext;
+import org.junit.Test;
+
+import java.sql.SQLException;
+
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class EncryptDQLResultDecoratorTest {
```
Comment (terrymanu) [inline shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-merge/src/test/java/org/apache/shardingsphere/encrypt/merge/dql/EncryptDQLResultDecoratorTest.java:32]: Please add final if the class is not design for extension

File: shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-merge/src/test/java/org/apache/shardingsphere/encrypt/merge/dql/EncryptDQLResultDecoratorTest.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    private EncryptAlgorithmMetaData metaData;
```
Comment (terrymanu) [inline shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-merge/src/test/java/org/apache/shardingsphere/encrypt/merge/dql/EncryptDQLResultDecoratorTest.java:33]: Please keep a blank line between class name and field decration.

**Review by terrymanu — CHANGES_REQUESTED** [review]
Why delete EncryptResultDecoratorEngineTest?

File: shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-merge/src/test/java/org/apache/shardingsphere/encrypt/merge/dql/EncryptDQLResultDecoratorTest.java (hunk 1 above)
Comment (dhc575757) [reply shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-merge/src/test/java/org/apache/shardingsphere/encrypt/merge/dql/EncryptDQLResultDecoratorTest.java:32]: > Why delete EncryptResultDecoratorEngineTest?
The code under over the wall is too slow on my side. Later, I downloaded the code with code cloud and uploaded the local change directly on GitHub. I found that the location was wrong, so I deleted it and added a new test class

Comment (dhc575757) [reply shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-merge/src/test/java/org/apache/shardingsphere/encrypt/merge/dql/EncryptDQLResultDecoratorTest.java:32]: > Please add final if the class is not design for extension

okay

**Pull request closed without merging**